hieradata_aws/training.yaml

```diff
@@ -196,6 +196,7 @@ users::usernames:
   - felisialoukou
   - fredericfrancois
   - ianjames
+  - isabelllong
```

Comment:
I don't think this environment exists at the moment, but there's no harm adding yourself.